homeassistant/components/media_player/strings.json

```diff
@@ -29,6 +29,124 @@
         "idle": "[%key:common::state::idle%]",
         "standby": "[%key:common::state::standby%]",
         "buffering": "Buffering"
+      },
+      "state_attributes": {
+        "app_id": {
+          "name": "App ID"
+        },
+        "app_name": {
+          "name": "App"
+        },
+        "entity_picture_local": {
+          "name": "Local accessible entity picture"
+        },
+        "groups_members": {
+          "name": "Group members"
+        },
+        "is_volume_muted": {
+          "name": "Muted",
+          "state": {
+            "true": "[%key:common::state::true%]",
+            "false": "[%key:common::state::false%]"
```

Comment:
Should we translate this to Yes/No ?

Reply:
does this even work, as attributes are boolean True/False, and this is a string value of true/false.

Reply:
Oh I see now that `true/false` placeholders translate to Yes/No. Not sure if we should do that, as it's not the same always, better to add a translation for Yes/No somewhere ?

Reply:
> Should we translate this to Yes/No ?

I think so? "Is it muted? Yes!" On/Off in this case isn't really fitting. True/False is IMHO too technical.

> does this even work, as attributes are boolean True/False, and this is a string value of true/false.

Yes, as they are converted to a translation string when looked up (template literal).

> Not sure if we should do that, as it's not the same always, better to add a translation for Yes/No somewhere ?

Added the generic state `yes`/`no` instead. Mapped true/false onto that.
Having them generic is helpful, as I have a few more PRs that will have a similar situation.